lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,34 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int msp_to_string(msParam_t* inp_param, char** out_param)
```

Comment:
Missing leading underscores on parameters.

Don't forget to update the log statements.